src/datasets/summarizations/trend-partial.summarization.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { queryFactory } from './trend-partial.summarization';
+import { TimeSeriesPoint } from '../metas/types';
+
+describe('queryFactory', () => {

```

Comment:
Would be great to have a couple tests for simple piecewise datasets. Perhaps a test for a "V" shaped dataset? Or even "W" shaped?